doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -445,6 +448,10 @@ Wi-Fi samples
 
   * :ref:`wifi_sr_coex_sample` sample demonstrating Wi-Fi Bluetooth LE coexistence.
 
+* Changed:
+
+  * The :ref:`wifi_shell_sample` sample now makes use of the :ref:`lib_wifi_credentials` and :ref:`wifi_mgmt_ext` libraries.
```

Comment:
```suggestion
  * The :ref:`wifi_shell_sample` sample now uses the :ref:`lib_wifi_credentials` and :ref:`wifi_mgmt_ext` libraries.
```

Reply:
Looks like you did not address this one.

Reply:
sorry, I meant to include it